### DISCUSSION TITLE
quiero un guia

### DISCUSSION THREAD

**Discussion by Yamil866** [community/community#48333] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

busco alguien que me guie como usar github y me enseñe como programar------favor de contactar amigos gracias 1

**Comment by valentinojosh** [comment]
¡Hola! Hay muchos recursos en línea gratuitos para aprender sobre GitHub y otros aspectos de la programación.

Aquí hay una lista que recomiendo:
-https://www.codecademy.com/
-https://github.com/free-courses/free-courses.github.io
-https://www.codecademy.com/learn/learn-git
-https://leetcode.com/
-https://www.hackerrank.com/

¡Espero que ayude!

**Comment by queenofcorgis** [comment]
Welcome to GitHub and the GitHub Community @Yamil866  🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
